sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -200,6 +203,29 @@ def _get_state_cache_size(pipeline_options):
   return 0
 
 
+def _get_data_buffer_time_limit_ms(pipeline_options):
+  """Defines the time limt of the outbound data buffering.
+
+  Note: data_buffer_time_limit_ms is an experimental flag and might
+  not be available in future releases.
+
+  Returns:
+    an int indicating the time limit in milliseconds of the the outbound
+      data buffering. Default is 0 (disabled)
+  """
+  experiments = pipeline_options.view_as(DebugOptions).experiments
+  experiments = experiments if experiments else []
+
+  for experiment in experiments:
+    # There should only be 1 match so returning from the loop
+    if re.match(r'data_buffer_time_limit_ms=', experiment):
+      return int(
+          re.match(
+              r'data_buffer_time_limit_ms=(?P<data_buffer_time_limit_ms>.*)',
```

Comment:
Should we be making sure that the experiments have the same string constant across languages ([beam_fn_api_data_buffer_time_limit](https://github.com/apache/beam/blob/4c18cb4ada2650552a0006dfffd68d0775dd76c6/sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/data/BeamFnDataBufferingOutboundObserver.java#L39))?

Reply:
I have also thought about this question when preparing this PR. The reason I have not done that in this PR is because I found that most config keys in the Java SDK harness starts with "beam_fn_api_" and it's not the same case for the config keys in the Python SDK harness.
 I'm fine to unify the config key if you don't think that's a problem. 
What's your thought? @lukecwik @mxm

Reply:
I think maintaining consistency in these options is important so that users have a good experience across languages. Changing the java ones to match the python ones would be fine as well since the java ones don't seem to follow an explicit naming convention.

Reply:
Make sense to me. I have created a PR https://github.com/apache/beam/pull/10505 as a follow-up. It changes the config key "beam_fn_api_data_buffer_time_limit" in the Java SDK harness to "data_buffer_time_limit_ms". Please note that the config key "beam_fn_api_data_buffer_size_limit" in the Java SDK harness is also changed to "data_buffer_size_limit" to have a consistent naming convension.

Reply:
+1 for unified experiment names across all languages.